packages/notebook-extension/src/index.ts

```diff
@@ -3357,6 +3359,29 @@ function addCommands(
     },
     isEnabled
   });
+  commands.addCommand(CommandIDs.toggleOutput, {
+    label: trans.__('Toggle Visibility of Selected Outputs'),
```

Comment:
Strictly: "of outputs of selected cell" (but this would be a bit too long probably)